pkgs/applications/misc/waybar/default.nix

```diff
@@ -65,28 +68,17 @@ stdenv.mkDerivation rec {
     )
   '';
 
-  nativeBuildInputs = [
-    meson ninja pkg-config scdoc wrapGAppsHook
-  ] ++ lib.optional withMediaPlayer gobject-introspection;
+  nativeBuildInputs = [ meson ninja pkg-config scdoc wrapGAppsHook systemd ]
+    ++ lib.optional withMediaPlayer gobject-introspection;
 
-  propagatedBuildInputs = lib.optionals withMediaPlayer [
-    glib
-    playerctl
-    python3.pkgs.pygobject3
-  ];
+  propagatedBuildInputs = lib.optionals withMediaPlayer [ glib playerctl python3.pkgs.pygobject3 ];
```

Comment:
We shouldn't change the formatting here, one item per line is actually better git diffable.

Reply:
That was just for consistency, should I then change all the other lists in this file?

Reply:
if that's important to you, yes